forui/lib/src/widgets/button/button_content.dart

```diff
@@ -0,0 +1,57 @@
+part of 'button.dart';
+
+/// Represents a content for [FButton].
+final class FButtonContent extends StatelessWidget {
+  /// The label on the button.
+  final String? text;
+
+  /// The icon;
+  final SvgAsset? icon;
+
+  /// The child.
+  final Widget? child;
+
+  /// The style.
+  final FButtonDesign style;
+
+  /// Whether the button should be disabled.
+  final bool disabled;
```

Comment:
This field should be called `enabled` instead to avoid double-negatives. Rmb to update the docs.